src/components/Checkout/Checkout.js

```diff
@@ -0,0 +1,487 @@
+import React, { useEffect,useState } from "react";
+import {
+  ArrowLeftOutlined,
+  ShoppingCartOutlined,
+  AlipayCircleOutlined,
+  CheckCircleOutlined,
+  DeleteOutlined,
+  LockOutlined,
+} from "@ant-design/icons";
+import "./checkout.css";
+import { Checkbox, Form, Input,notification,Steps } from "antd";
+import { Link } from "react-router-dom";
+import axios from "axios";
+import { useDispatch, useSelector } from "react-redux";
+import { setAddress } from "../../redux/actions/productActions";
+
+const { Step } = Steps;
+
+const Checkout = () => {
+  const [form] = Form.useForm();
+  const [oneAddress, setOneAddress] = useState({});
+  const [toggleButton, setToggleButton] = useState(false);
+  const [coin, setCoin] = useState(0);
+
```

Comment:
1. const  [state,setState] = useState({coin:0,toggleButton:false});
2. if you have more than 3 state : useReducer

Reply:
Done